### PR TITLE
feat(aws): Add command to create cross account role via CloudFormatio…

### PR DESCRIPTION
**Pull request by th3morg**
…n stack

## How has this been tested? How can a reviewer test these changes?

# Have your performed the following tests?

Please confirm you have done the following tests to ensure your pull request is ready to be merged..

- [ ] If this change modifies the deployment flow have triggered a deployment using this branch (i.e. `build/dist/darwin_amd64/armory deploy start -f <path to file>`)?
- [ ] If this change modifes the login flow have you tried logging in and out with this branch (i.e. `build/dist/darwin_amd64/armory login`) and verified the credentials work?
- [ ] Have you verified the specific change made in this PR?

### CODE REVIEW

File: cmd/config/aws/aws.go [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+const (
+	configAWSShort = "Manage CD-as-a-Service AWS Access"
+	configAWSLong  = "Manage CD-as-a-Service AWS Access\n\n" +
+		"For usage documentation, visit TODO"
+	configAWSExample = "armory aws <subcommand>"
+)
+
+func NewAWSCmd(configuration *cliconfig.Configuration) *cobra.Command {
+	command := &cobra.Command{
```
Comment (danielpeach) [inline cmd/config/aws/aws.go:18]: maybe keep the command hidden for now, especially if there's a TODO in the help fields?

File: cmd/config/aws/create_role_cmd.go [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+	log "go.uber.org/zap"
+	"io"
+)
+
+const (
+	configCreateRoleShort = "AWS Create Role via CloudFormation Stack"
+	configCreateRoleLong  = "Use AWS CloudFormation quick create to run a template that allows Armory to assume a role to manage deployments\n\n" +
+		"For usage documentation, visit TODO"
+	configCreateRoleExample = "armory config aws create-role"
+	templateUrl             = "https://us-east-1.console.aws.amazon.com/cloudformation/home?region=us-east-1#/stacks/quickcreate?templateURL=https://armory-cdaas-cloudformation.s3.us-west-2.amazonaws.com/iam-role-cfn.template&stackName=Armory-CDAAS-Role-Stack&param_AccountId=%s&param_ExternalId=%s"
```
Comment (danielpeach) [inline cmd/config/aws/create_role_cmd.go:21]: should this be different for different Armory environments (e.g., what if I want to try the whole Lambda setup workflow against staging)?

File: cmd/config/aws/create_role_cmd.go [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+	return cmd
+}
+
+func createRole(cmd *cobra.Command, cli *cliconfig.Configuration) error {
+	orgID, err := cli.GetAuth().GetOrganizationId()
+	if err != nil {
+		return auth.ErrNotLoggedIn
+	}
+
+	envID, err := cli.GetAuth().GetEnvironmentId()
```
Comment (danielpeach) [inline cmd/config/aws/create_role_cmd.go:47]: this error is ignored, though it's unlikely that the first one would be nil and the second one wouldn't

File: cmd/config/aws/create_role_cmd.go [hunk 4]
```diff
@@ -0,0 +1,76 @@
...
+	externalID := fmt.Sprintf("%s/%s", orgID, envID)
```
Comment (danielpeach) [inline cmd/config/aws/create_role_cmd.go]: we always represent it as `orgID:envID` (with a colon). Is there some restriction in the external ID charset that prevents us from doing it that way?

File: cmd/config/aws/create_role_cmd.go [hunk 5]
```diff
@@ -0,0 +1,76 @@
...
+	log.S().Info("2. Click \"Create\" in the AWS CloudFormation screen and wait for the stack creation to complete.")
+	log.S().Info("3. Once the CloudFormation is finished, locate the ArmoryRole by navigating to IAM > Roles in the AWS Management console. Use the `arn` value as the `account` in your CD-as-a-Service deployment targets.")
+	prompt := promptui.Prompt{
+		Label:     "Continue",
+		IsConfirm: true,
+		Default:   "Y",
+		Stdout:    &util.BellSkipper{},
+	}
+
+	_, err = prompt.Run()
```
Comment (danielpeach) [inline cmd/config/aws/create_role_cmd.go]:
```go
if _, err := prompt.Run(); err != nil {
  return err
}
```

File: cmd/config/aws/create_role_cmd_test.go [hunk 6]
```diff
@@ -0,0 +1,31 @@
...
+)
+
+func getStdTestConfig(token string, outFmt string) *config.Configuration {
+	addr := "https://localhost"
+	clientId := ""
+	clientSecret := ""
+	isTest := true
+	return config.New(&config.Input{
+		AccessToken:  &token,
+		ApiAddr:      &addr,
```
Comment (danielpeach) [inline cmd/config/aws/create_role_cmd_test.go]: I prefer doing `lo.ToPtr("https://localhost")` these days but it's up to you.

File: cmd/config/aws/create_role_cmd.go (hunk 2 above)
Comment (danielpeach) [reply cmd/config/aws/create_role_cmd.go:21]: nvm, you did do that! ignore me.

File: cmd/config/aws/create_role_cmd.go (hunk 4 above)
Comment (th3morg) [reply cmd/config/aws/create_role_cmd.go]: Nothing in particular. Will change to colon

File: cmd/config/aws/create_role_cmd_test.go (hunk 6 above)
Comment (th3morg) [reply cmd/config/aws/create_role_cmd_test.go]: Much better - agreed :)

**Review by danielpeach — APPROVED** [review]

**Pull request merged**